src/anchorfm-pupeteer/index.js

```diff
@@ -135,6 +135,12 @@ async function postEpisode(youtubeVideoInfo) {
     } else {
       await spotifyLogin();
     }
+    try {
+      await page.waitForNavigation();
+    } catch (err) {
+      /* empty */
```

Comment:
Let's just log here that the wait for navigation failed, so we can understand from the log a bit more

Reply:
Thanks for the suggestion. Done.